members/lisa_je.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+---
+
+<!-- Replace `example_student` with your name -->
+<img src="/website/assets/img/lisa_je.png" alt="Placeholder Image" class="center" style="max-width: 100%">
+
+<!-- Replace `Example Student` with your name and include your start date-->
+# **Lisa Je (2023 - present)**
+
+<!-- Choose your title -- feel free to be professionally silly -->
+**Role in the Cersonsky Lab**: Collaborator
```

Comment:
```suggestion
**Role in the Cersonsky Lab**: Unofficial Senior Grad Student
```

Reply:
I'm going to keep it as collaborator to be "politically correct" amongst groups (and also I committed already)